server/api/controllers/asgs/asgController.js

```diff
@@ -116,12 +140,26 @@ function putAsg(req, res, next) {
 
   const parameters = req.swagger.params.body.value;
 
-  UpdateAutoScalingGroup({ environmentName, autoScalingGroupName, parameters })
-    .then(data => res.json(data)).catch(next);
+  UpdateAutoScalingGroup({
+    environmentName,
+    autoScalingGroupName,
+    parameters
+  })
+    .then(data => res.json(data))
+    .then(sns.publish({
```

Comment:
What would the topic look like for this? It doesn't seem to mention that it's a change to an ASG?